keanu-project/src/test/java/io/improbable/keanu/vertices/booltensor/nonprobabilistic/operators/multiple/BooleanMultipleOperatorTest.java

```diff
@@ -0,0 +1,65 @@
+package io.improbable.keanu.vertices.booltensor.nonprobabilistic.operators.multiple;
+
+import io.improbable.keanu.tensor.bool.BooleanTensor;
+import io.improbable.keanu.vertices.Vertex;
+import io.improbable.keanu.vertices.booltensor.BoolVertex;
+import io.improbable.keanu.vertices.booltensor.probabilistic.Flip;
+import io.improbable.keanu.vertices.dbltensor.KeanuRandom;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class BooleanMultipleOperatorTest {
+
+    private List<Vertex<BooleanTensor>> allTrue = buildVertexList(15, 15);
+    private List<Vertex<BooleanTensor>> allMixed = buildVertexList(18, 10);
+    private List<Vertex<BooleanTensor>> allFalse = buildVertexList(15, 0);
+    private KeanuRandom random;
+
+    @Before
+    public void setup() {
+        this.random = new KeanuRandom(1);
+    }
+
+    @Test
+    public void testMultipleAnd() {
+        BoolVertex andTrue = new AndMultipleVertex(allTrue);
+        assertEquals(true, andTrue.sample(random).scalar());
```

Comment:
We need to think about a nice API for doing a shorthand

`.sample(random).scalar()`
or
`.getValue().scalar()`